src/main/java/racingcar/view/OutputView.java

```diff
@@ -0,0 +1,63 @@
+package racingcar.view;
+
+import java.util.List;
+import racingcar.domain.car.Car;
+import racingcar.domain.car.Cars;
+
+public class OutputView {
+
+    private OutputView() {
+    }
+
+    public static OutputView getInstance() {
+        return new OutputView();
+    }
+
+    public void printCarNamesInput() {
+        System.out.println(Message.INPUT_CAR_NAMES.getMessage());
+    }
+
+    public void printAttemptsInput() {
+        System.out.println(Message.INPUT_ATTEMPTS_NUMBER.getMessage());
+    }
+
+    public void printExecutionResult() {
+        System.out.printf(Message.EXECUTION_RESULT.getMessage());
+    }
+
+    public void printResult(Cars cars) {
+        for (Car car : cars.getCars()) {
+            System.out.printf(Message.RESULT_FORM.getMessage(), car.getName(),
+                    Message.POSITION_MARK.getMessage().repeat(car.getPosition()));
```

Comment:
```suggestion
                    Message.POSITION_MARK
                    .getMessage()
                    .repeat(car.getPosition()));
```
개행을 통해 가독성을 높여주시는게 좋을 것 같습니다!